vocabularies/UI.xml

```diff
@@ -1554,9 +1554,15 @@ It is NOT tied to a data value (in contrast to [DataFieldWithIntentBasedNavigati
 
       <ComplexType Name="DataField" BaseType="UI.DataFieldAbstract">
         <Annotation Term="Core.Description" String="A piece of data" />
-        <Property Name="Value" Type="Edm.PrimitiveType" Nullable="false">
+        <Property Name="Value" Type="Edm.Untyped" Nullable="false">
           <Annotation Term="Core.Description" String="The data field's value" />
           <Annotation Term="Core.IsLanguageDependent" />
+          <Annotation Term="Validation.DerivedTypeConstraint">
+            <Collection>
+              <String>Edm.PrimitiveType</String>
+              <String>Collection(Edm.PrimitiveType)</String>
+            </Collection>
+          </Annotation>
```

Comment:
Is `Collection(Edm.PrimitiveType)` a qualified type name?
https://github.com/oasis-tcs/odata-vocabularies/blob/32ec59be5f831018d5fc40ac46288b017a889faf/vocabularies/Org.OData.Core.V1.xml#L452-L454

Or should the type definition say "... or a collection thereof"?